monix-reactive/shared/src/test/scala/monix/reactive/internal/operators/PublishSelectorSuite.scala

```diff
@@ -32,7 +32,7 @@ object PublishSelectorSuite extends BaseTestSuite {
       .publishSelector { source =>
         Observable(source, source, source).merge
       }
-      .sumL
+      .sumL[Long]
```

Comment:
There is this error (doesn't happen for `Int` ) without type ascription:

```
|ambiguous implicit arguments: both object IntIsIntegral in object Numeric and object DoubleIsFractional in object Numeric match type Numeric[B] of parameter B of method sumL in class Observable
```

didn't have time to investigate why